pkg/controllers/machinesync/machine_sync_controller.go

```diff
@@ -99,7 +111,80 @@ func (r *MachineSyncReconciler) SetupWithManager(mgr ctrl.Manager) error {
 }
 
 // Reconcile reconciles CAPI and MAPI machines for their respective namespaces.
+//
+//nolint:funlen
 func (r *MachineSyncReconciler) Reconcile(ctx context.Context, req reconcile.Request) (ctrl.Result, error) {
+	logger := log.FromContext(ctx, "namespace", req.Namespace, "name", req.Name)
+
+	logger.V(1).Info("Reconciling machine")
+	defer logger.V(1).Info("Finished reconciling machine")
+
+	var (
+		mapiMachineNotFound, capiMachineNotFound bool
+	)
```

Comment:
Does the following not work? Wonder why you put it in parentheses?
```suggestion
	var mapiMachineNotFound, capiMachineNotFound bool
```

Reply:
Ah, I'd initially done them over two lines - didn't remember the parentheses